features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -131,6 +131,13 @@
     </OpenID>
 
     <OAuth>
+        <!-- token cleanup feature config-->
```

Comment:
Minor: Capitalize starting letter. Also better to describe which table is cleaned up in this manner.